python/dftd4/test_ase.py

```diff
@@ -79,5 +88,11 @@ def test_ase_tpssd4():
 
     atoms.calc = DFTD4(method="TPSS").add_calculator(EMT())
     assert approx(atoms.get_potential_energy(), abs=thr) == 4.864016486351274
-    energies = [calc.get_potential_energy() for calc in atoms.calc.calcs]
+
+    if ase_version >= (3, 23, 0):
```

Comment:
We could check here if the mixer attr exists instead of relying on version numbers